static/js/giphy.js

```diff
@@ -99,6 +112,48 @@ function update_grid_with_search_term() {
     return undefined;
 }
 
+export function hide_giphy_popover() {
+    // Returns `true` if the popover was open.
+    if (active_popover_element) {
+        active_popover_element.popover("hide");
+        active_popover_element = undefined;
+        edit_message_id = undefined;
+        return true;
+    }
+    return false;
+}
+
+function get_popover_template() {
+    let template = render_giphy_picker();
+
+    if (window.innerWidth <= Number(media_breakpoints.md_min.slice(0, -2))) {
+        // Show as modal in the center for small screens.
+        template = render_giphy_picker_mobile();
+    }
+
+    return template;
+}
```

Comment:
This is cleaner written this way, since we avoid doing the rendering work twice on mobile screen sizes and also the `template` variable (which should probably be called `rendered_picker`, not "template", because an "HTML template" is the source file that generates "rendered HTML") ; I'll just fix when merging this commit.

```
diff --git a/static/js/giphy.js b/static/js/giphy.js
index c49885ebd7..170c2678bd 100644
--- a/static/js/giphy.js
+++ b/static/js/giphy.js
@@ -124,14 +124,11 @@ export function hide_giphy_popover() {
 }
 
 function get_popover_template() {
-    let template = render_giphy_picker();
-
     if (window.innerWidth <= Number(media_breakpoints.md_min.slice(0, -2))) {
         // Show as modal in the center for small screens.
-        template = render_giphy_picker_mobile();
+        return render_giphy_picker_mobile();
     }
-
-    return template;
+    return render_giphy_picker();
 }
 
 function get_popover_placement() {
```

Also renamed to `get_popover_content`, for the same naming reason.